scripts/qemu-dm-wrapper

```diff
@@ -19,6 +19,8 @@ path = "/sbin:/usr/sbin:/bin:/usr/bin"
 
 xenstore_write_cmd = "xenstore-write"
 
+cgroup_slice = ''
```

Comment:
OK, I see the directory structure under `/sys/fs/cgroup/cpu/` now, but that was not obvious. Perhaps a comment would help to explain this...